project/profiles/capacity4more/modules/c4m/field/c4m_field_og/c4m_field_og.module

```diff
@@ -119,7 +119,10 @@ function c4m_field_og_field_access($op, $field, $entity_type, $entity, $account)
  * Hides certain status values at form, according to users access rights.
  */
 function c4m_field_og_form_group_node_form_alter(&$form, &$form_state, $form_id) {
-  _c4m_field_og_set_status_flow($form, $form_state, 'group');
+  // If editing node, set the status flow.
+  if (isset($form['nid']['#value'])) {
```

Comment:
@anvmn 
what's the problem in here ? the comment is lacking ...

ps
not sure how's the group content type is related to this PR.